src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/w3c/JavascriptW3cPerformanceTimeline.kt

```diff
@@ -1,117 +1,31 @@
 package com.atlassian.performance.tools.jiraactions.api.w3c
 
+import com.atlassian.performance.tools.jiraactions.api.w3c.harvesters.getJsElementsPerformance
+import com.atlassian.performance.tools.jiraactions.api.w3c.harvesters.getJsNavigationsPerformance
+import com.atlassian.performance.tools.jiraactions.api.w3c.harvesters.getJsResourcesPerformance
 import org.openqa.selenium.JavascriptExecutor
-import java.time.Duration
 
 /**
  * Obtains entries from [javascript].
  */
 class JavascriptW3cPerformanceTimeline(
-    private val javascript: JavascriptExecutor
+    javascript: JavascriptExecutor,
+    withNavigationPerformance: Boolean = true,
+    withResourcesPerformance: Boolean = true,
+    withElementPerformance: Boolean = true
```

Comment:
Kotlin default arguments are a trap. They use compilation tricks, which leave messy API, which cannot be cleanly used from Java.
We need to replace this constructor with a builder. A builder is a good vessel for default values.

Reply:
Makes sense!
- [ ] Do it!